miasm2/ir/ir.py

```diff
@@ -278,25 +300,34 @@ def _get_dst(self):
         self._dst_linenb = final_linenb
         return final_dst
 
-    def _set_dst(self, value):
-        """Find and replace the IRDst affectation's source by @value"""
-        if self._dst_linenb is None:
-            self._get_dst()
+    @property
+    def dst(self):
+        """Find the IRDst affectation and update dst, dst_linenb accordingly"""
```

Comment:
For me, the docstring is misleading here: this is more a dev comment than the description of what `.dst` is, that is to say the destination of the block as an `Expr` instance (external user does not have to care about cache, especially in the case of immutable object).